e2e/integration/homePage.js

```diff
@@ -0,0 +1,56 @@
+context( 'Test homepage', () => {
+    before( () => {
+        cy.visit( Cypress.env( 'STAGE_HOST' ) );
+        Cypress.config( 'includeShadowDom', true );
+        cy.get( '#username', { timeout: 5000 } ).type( Cypress.env('USERNAME') );
+        cy.get( '#password' ).type( Cypress.env( 'PASSWORD' ) );
+        cy.get( '#submit' ).click();
+    } );
+
+    it( 'Test for home page header section', () => {
+        cy.get( '.hero__mantra' ).should( 'contain.text', 'Develop Fast·Deliver Faster' );
+        cy.get( '.hero__quote' ).should( 'contain.text', 'One Platform to host your Internal Applications and Services' );
+        cy.get( '.hero__description' ).should( 'contain.text', 'One Portal provides a single place for all internal applications and services, supports consistent User experience by providing standard platform for service hosting and data integration, efficient resource management, real time metrics availability, cross-team collaboration and unified documentation.' );
+        cy.get( '.button--danger' ).should( 'contain.text', 'View Applications' );
+        cy.get( '.button--light' ).should( 'contain.text', 'Built-in Microservices' );
+    } );
+
+    it( 'Test for apps list section', () => {
+        cy.get( '.op-menu-drawer__app-list-item ' ).each( () => {
+            cy.get( 'a[ href = "/outages" ]' ).should( 'be.visible' );
+            cy.get( 'a[ href = "#" ]' ).should( 'be.visible' );
```

Comment:
We should maintain a list of apps deployed somewhere in JSON file so that if a new app is deployed we don't need to update the actual test files. We just update the JSON file and tests should work.